tasks/main.py

```diff
@@ -273,6 +273,8 @@ def main(ctx):
                 run_task(ctx, 'build-static',
                          log_callback=LOG_CALLBACK,
                          private_values=private_values)
+            elif GENERATOR == 'script only':
+                LOGGER.info('build already ran in \'npm run federalist\'')
```

Comment:
Is the assumption that they will build their site into `_site`?

Reply:
yes

Reply:
We'll put that into federalist-docs though. Should we also link to that page from the "settings" and "new sites" screens?